lib/heroku/command/pg.rb

```diff
@@ -463,7 +463,9 @@ def maintenance
 
   # pg:upgrade REPLICA
   #
-  # unfollow a database and upgrade it to the latest PostgreSQL version
+  # unfollow a database and upgrade it to the latest, stable PostgreSQL version
```

Comment:
I'm no grammar expert, but 'latest' modifies 'stable' here, so I don't think you need a comma...